### PR TITLE
displayed subjects in academicInfo ui

### PR DESCRIPTION
**Pull request by Richmond-Andoh**

### CODE REVIEW

File: CorsaSchoolManagementSystem/studentDetails.php [hunk 1]
```diff
@@ -6,6 +6,8 @@
     $users = new User(new Database());
     $template = new Template("Views/studentDetails.php");
     $studentDetails = $users->getStudentDetails(htmlspecialchars($_GET['studentId']));
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/studentDetails.php]: You can get the subjects from studentDetails, hence, no need to call the getClasses method. 
Check the getStudentDetails() query, you will notice the subjects have been fetched already

**Review by OliverMensahDev — COMMENTED** [review]
Also, make sure you don't push any images you have uploaded while you were testing locally to the main codebase. 

For the meantime, remove those images you have included

**Comment by Richmond-Andoh** [comment]
Okay

**Comment by OliverMensahDev** [comment]
Is this how you want to display them?

Screenshot 2022-12-23 at 9 44 17 PM

**Comment by Richmond-Andoh** [comment]
Not really I'm finding a better way to display them

**Pull request closed without merging**